crates/cairo-lint-core/tests/test_files/loop_for_while/loop_for_while

```diff
@@ -139,13 +175,36 @@ warning: Plugin diagnostic: you seem to be trying to use `loop`. Consider replac
 //! > fixed
 fn main() {
     let mut value: u16 = 100;
-    while !(value < 0) {
+    while value >= 0 {
         value -= 10;
     }
 }
 
 //! > ==========================================================================
 
+//! > loop with nested condition inside if block
```

Comment:
Nested is a special word that will not trigger the fix. Only use it if you have multiple diagnostics on the same lines for example if you have 2 nested while instead of loop (not here)
```suggestion
//! > loop with multiple condition inside if block
```